lib/api/status.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-function configure (app, wares, env) {
+function configure (app, wares, ctx, env) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["env" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1885925981/issues/source?bid=2452379&fileBranchId=3084282#l3)